File: panda-ij-plugin/src/eu/nyerel/panda/ijplugin/runner/calltree/CallTreeToolWindowCondition.java
package eu.nyerel.panda.ijplugin.runner.calltree;

import com.intellij.openapi.project.Project;
import com.intellij.openapi.util.Condition;

/**
 * @author Rastislav Papp (rastislav.papp@gmail.com)
 */
public class CallTreeToolWindowCondition implements Condition<Project> {

    @Override
    public boolean value(Project project) {
        return false;
    }

}

File: panda-ij-plugin/src/eu/nyerel/panda/ijplugin/runner/calltree/CallTreeToolWindowCondition.kt
package eu.nyerel.panda.ijplugin.runner.calltree

import com.intellij.openapi.project.Project
import com.intellij.openapi.util.Condition

/**
 * @author Rastislav Papp (rastislav.papp@gmail.com)
 */
class CallTreeToolWindowCondition : Condition<Project> {

    override fun value(project: Project): Boolean {
        return true
    }

}
